conda/dev.yaml

```diff
@@ -1,9 +1,12 @@
 name: alertadengue
 channels:
   - conda-forge
-  - nodefaults
```

Comment:
You probably should keep nodefaults to avoid mixing packages from anaconda and conda forge